pkg/config/config.go

```diff
@@ -2,12 +2,14 @@ package config
 
 import (
 	"fmt"
+	"log/slog"
 	"sort"
 	"strings"
 	"sync"
 	"time"
 
-	log "github.com/sirupsen/logrus"
+	"github.com/flant/shell-operator/pkg/unilogger"
+	log "github.com/flant/shell-operator/pkg/unilogger"
```

Comment:
And again the same package under two names, why?

Reply:
fixed